pyfr/solvers/euler/inters.py

```diff
@@ -15,6 +15,11 @@ def __init__(self, *args, **kwargs):
                 entmin_lhs=self._entmin_lhs, entmin_rhs=self._entmin_rhs
             )
 
+            self.p_min = self.cfg.getfloat('solver-entropy-filter', 'p_min',
+                                           1e-6)
+        else:
+            self.p_min = self._be.fpdtype_eps
```

Comment:
Two things: 1) I think this should be a parameter open in the solver-interfaces section (but not documented kind of like the entropy filter iterative solver parameters) so that it can be modified by people who know what they're doing if you're not using the entropy filter. 2) I think the default should be at least 2*eps or some factor of eps (maybe 10?), otherwise it seems too prone to roundoff error.